code/game/turfs/hybrisa.dm

```diff
@@ -0,0 +1,717 @@
+	icon_state = "engineer_floor_corner3"
+
+// Pillars
+
+/turf/open/floor/hybrisa/engineership/pillars
+	name = "strange metal pillar"
+	desc = "A strange metal pillar, unlike any metal you've seen before."
+	icon_state = "eng_pillar1"
+
+/turf/open/floor/hybrisa/engineership/pillars/north/pillar1
+	icon_state = "eng_pillar1"
+/turf/open/floor/hybrisa/engineership/pillars/north/pillar2
+	icon_state = "eng_pillar2"
+/turf/open/floor/hybrisa/engineership/pillars/north/pillar3
+	icon_state = "eng_pillar3"
+/turf/open/floor/hybrisa/engineership/pillars/north/pillar4
+	icon_state = "eng_pillar4"
+/turf/open/floor/hybrisa/engineership/pillars/south/pillarsouth1
+	icon_state = "eng_pillarsouth1"
+/turf/open/floor/hybrisa/engineership/pillars/south/pillarsouth2
+	icon_state = "eng_pillarsouth2"
+/turf/open/floor/hybrisa/engineership/pillars/south/pillarsouth3
+	icon_state = "eng_pillarsouth3"
+/turf/open/floor/hybrisa/engineership/pillars/south/pillarsouth4
+	icon_state = "eng_pillarsouth4"
+/turf/open/floor/hybrisa/engineership/pillars/west/pillarwest1
+	icon_state = "eng_pillarwest1"
+/turf/open/floor/hybrisa/engineership/pillars/west/pillarwest2
+	icon_state = "eng_pillarwest2"
+/turf/open/floor/hybrisa/engineership/pillars/west/pillarwest3
+	icon_state = "eng_pillarwest3"
+/turf/open/floor/hybrisa/engineership/pillars/west/pillarwest4
+	icon_state = "eng_pillarwest4"
+/turf/open/floor/hybrisa/engineership/pillars/east/pillareast1
+	icon_state = "eng_pillareast1"
+/turf/open/floor/hybrisa/engineership/pillars/east/pillareast2
+	icon_state = "eng_pillareast2"
+/turf/open/floor/hybrisa/engineership/pillars/east/pillareast3
+	icon_state = "eng_pillareast3"
+/turf/open/floor/hybrisa/engineership/pillars/east/pillareast4
+	icon_state = "eng_pillareast4"
+
+
+// -------------------- // Hybrisa Wall types // ---------------- //
+
+// Derelict Ship
+
+/turf/closed/wall/engineership
+	name = "strange metal wall"
+	desc = "Nigh indestructible walls that make up the hull of an unknown ancient ship."
+	icon = 'icons/turf/walls/engineership.dmi'
+	icon_state = "metal"
+	walltype = WALL_HUNTERSHIP
+	hull = TRUE
+
+/turf/closed/wall/engineership/destructible
+	name = "strange metal wall"
+	desc = "Nigh indestructible walls that make up the hull of an unknown ancient ship."
+	icon = 'icons/turf/walls/engineership.dmi'
+
+	walltype = WALL_HUNTERSHIP
+	hull = 0
+	damage_cap = HEALTH_WALL_ULTRA_REINFORCED
+	baseturfs = /turf/open/floor/plating/engineer_ship
+
+// Rock
+
+/turf/closed/wall/hybrisa/rock
+	name = "rock wall"
+	desc = "Massive columns comprised of anicent sedimentary rocks loom before you."
+	icon = 'icons/turf/walls/kutjevorockdark.dmi'
+	icon_state = "rock"
+	walltype = WALL_KUTJEVO_ROCK
+	hull = TRUE
+
+// Marshalls
+
+/turf/closed/wall/hybrisa/marhsalls
+	name = "metal wall"
+	icon = 'icons/turf/walls/hybrisa_marshalls.dmi'
+	icon_state = "metal"
+	walltype = WALL_METAL
+/turf/closed/wall/hybrisa/marhsalls_reinforced
```

Comment:
Any reason why not just `/turf/closed/wall/hybrisa/marhsalls/reinforced` so you can just use inheritance and only change what needs to change?